org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: 3.30
+sdk: org.gnome.Sdk
+rename-icon: gaphor
+command: gaphor
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconfig:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+cleanup:
+  - /include
+  - /lib/debug
+modules:
+  - generated-poetry-sources.json
+  - name: gaphor
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --disable-pip-version-check --no-build-isolation --no-deps .
+    sources:
+      - type: git
+        tag: 1.0.0rc1
+        url: https://github.com/gaphor/gaphor
+        commit: 48c2bcff5ae32a6b6c4cae19d8d43634573741de
+    post-install:
+      - install -Dm644 flatpak/org.gaphor.Gaphor.appdata.xml /app/share/appdata/org.gaphor.Gaphor.appdata.xml
+      - install -Dm644 flatpak/org.gaphor.Gaphor.desktop /app/share/applications/org.gaphor.Gaphor.desktop
```

Comment:
As you're upstream, it would be nice if you could install the appdata/desktop files & the icons using the buildsystem you're using currently. 
Also the icons should be named `org.gaphor.Gaphor` and you can ditch the `rename-icon`